plugins/generator-1.19.2/forge-1.19.2/templates/json/item_override.json.ftl

```diff
@@ -0,0 +1,6 @@
+{

```

Comment:
Why are those overrides needed? You could use existing templates for this and make them fallback to different variable if data is not defined

Reply:
You mean like this?
`<#assign model = (item??)then(item, data)>`
And then replace all `data` with `model`/`source`.

Reply:
Yes